src/PrestaShopBundle/Entity/Translation.php

```diff
@@ -36,6 +37,7 @@
  *     indexes={@ORM\Index(name="key", columns={"domain"})},
  * )
  * @ORM\Entity(repositoryClass="PrestaShopBundle\Entity\Repository\TranslationRepository")
+ * @PassVsprintf
```

Comment:
I could not make this annotation work, even if I followed the documentation (http://symfony.com/doc/2.8/validation/custom_constraint.html).

I manually trigger the validation in the controller.